packages/sui-bundler/plugins/loader-options.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Simple plugin that guarantees loaders to be loaded with the given options
+ * in any context.
+ * This fixes a bug of webpack that was loading files outside of context with
+ * the given loaders, but not their options.
+ *
+ * @param {Object} [options={}] key = loader name, value = loader options
+ */
+function LoaderUniversalOptionsPlugin (options = {}) {
+  this.loaderOptions = options
+}
+
+LoaderUniversalOptionsPlugin.prototype.apply = function (compiler, compilation) {
```

Comment:
Don't know how to optimize it. Take into account the loader name is converted to loader path by webpack.
And the path is different denpending on context (sui-studio vs sui-bundler)